src/com/lftechnology/java_training/dec31/EmployeeTest.java

```diff
@@ -0,0 +1,123 @@
+package com.lftechnology.java_training.dec31;
+
+import java.util.Scanner;
+
+/**
+ * Write a test application named EmployeeTest that demonstrates class
+ * Employee's capabilities. Create two Employee objects and display each
+ * object's full name and yearly salary. Then give each Employee a 10% raise and
+ * display each Employee's full name and yearly salary again.
+ * 
+ * @author utsabn34
+ * 
+ */
+public class EmployeeTest {
+
+	public static void main(String[] args) {
+		Employee employee1 = new Employee();
+		Employee employee2 = new Employee();
+		employee1.assignData("employee1");
+		employee2.assignData("employee2");
+		employee1.displayData();
+		employee2.displayData();
+		System.out.println("After 10% rise in Salary:");
+		employee1.setData(employee1.getFirstName(), employee1.getLastName(),
```

Comment:
IS this necessary to set again firstname and last name in the setData method, since the first and last name already has been set in the method assignData()